### PR TITLE
changed headline

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: create_framework/index.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Create your PHP Framework
+Create your own PHP Framework
 =========================
```
Comment (javiereguiluz) [inline create_framework/index.rst]: Minor issue: in Symfony docs, the second line of the headline must be of the same length of the first line.

**Comment by OskarStark** [comment]
fixxed @javiereguiluz

**Comment by OskarStark** [comment]
arrgh :)

**Comment by OskarStark** [comment]
now.... grml :)

**Comment by xabbuh** [comment]
:+1: looks good

**Comment by WouterJ** [comment]
Thank you Oskar.

**Pull request closed without merging**